src/app/tenant/pages/tenant-search/tenant-search.component.ts

```diff
@@ -79,27 +81,53 @@ export class TenantSearchComponent implements OnInit {
         routerLink: '/tenant'
       }
     ])
+
+    this.store
+      .select(tenantSearchSelectors.selectCriteria)
+      .subscribe((criteria) => this.tenantSearchFormGroup.reset(criteria))
+  }
+
+  searchConfigInfoSelectionChanged(
+    searchConfig:
+      | {
+          fieldValues: Record<string, string>
+          displayedColumnsIds: string[]
+          viewMode: 'basic' | 'advanced'
+        }
+      | undefined
+  ) {
+    if (searchConfig) {
+      this.store.dispatch(
```

Comment:
Dispatching of actions is never done conditionally because the event happened the code which decides if something is done as a result of the event is somewhere else